Client/src/Components/Sidebar/index.jsx

```diff
@@ -71,6 +73,8 @@ const menu = [
 const other = [
   { name: "Support", path: "support", icon: <Support /> },
   { name: "Settings", path: "settings", icon: <Settings /> },
```

Comment:
Could you move `Settings` to the first position here?  I think that makes more sense since users will probably use Settings more than any of the other links.